src/components/layouts/Header.js

```diff
@@ -10,6 +12,7 @@ import PropTypes from 'prop-types';
 import logo from '../../assets/img/logo_st.png';
 import { initProfile } from '../../redux/actions/profile.actions';
 import Profilemenu from '../menu/Profile';
+import { getUnreadNotifications, readNotification } from '../../redux/actions/notifications.actions';
 // import { setLoggedIn } from '../../redux/actions/ui.actions';
```

Comment:
Please delete this commented line

Reply:
it's not my line but I am going to delete it!

Reply:
Thank you @mifeille